cypress/integration/capture.spec.js

```diff
@@ -1,7 +1,10 @@
 /// <reference types="cypress" />
+import { version } from '../../package.json'
 
 import { getBase64EncodedPayload, getGzipEncodedPayload, getLZStringEncodedPayload } from '../support/compression'
 
+const urlWithLibraryAndVersion = new RegExp(`&l=web&v=${version}`)
```

Comment:
I think `l=web&v=` is a pretty uniquely matchable pattern for aggressive traffic blockers, so perhaps we should infer `web` by default if only `v` is present? Other integrations than posthog-js can send it if they want, but there are benefits to not exposing it here.